mount/mount_windows.go

```diff
@@ -9,15 +16,68 @@ var (
 
 // Mount to the provided target
 func (m *Mount) Mount(target string) error {
-	return ErrNotImplementOnWindows
+	home, layerID := filepath.Split(m.Source)
```

Comment:
nit: this can be done after `m.GetParentPaths()` to avoid allocating the memory if not necessary